Remove the out-of-date banner on the MiniKF on GCP page

cspavlou opened the pull request:
- Remove the out-of-date banner, as MiniKF on Google Cloud uses the latest manifests
- Update some screenshots to to reflect the latest MiniKF version on Google Cloud
- Replace all occurrences of "GCP" with "Google Cloud"

8bitmp3 commented on content/en/docs/started/workstation/minikf-gcp.md:
```diff
@@ -3,12 +3,6 @@ title = "Deploy using MiniKF on GCP"
 description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
```
Thanks @cspavlou !

A few suggestions, since we're in this guide:

- Use "Google Cloud" (rebranding?) instead of "...Platform"
- Change the order of the words in the description

```suggestion
description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via Google Cloud Marketplace"
```

cc @alfsuse @RFMVasconcelos 

Also, in the title (couldn't highlight directly):

- Let's be a bit more descriptive

```diff
- title = "Deploy using MiniKF on GCP"		
+ title = "Deploy Kubeflow using MiniKF on Google Cloud"
```

LMKWYT

8bitmp3 replied:
Line 16 and elsewhere: "Google Cloud" <- "GCP", if possible

cspavlou replied:
Thanks for your comments! I will update accordingly

8bitmp3 reviewed (changes requested):
Thanks @cspavlou !

A few suggestions, since we're in this guide:

- Use "Google Cloud" (rebranding?) instead of "...Platform"
- Change the order of the words in the description

```diff
- description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
+ description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via Google Cloud Marketplace"
```

Also, in the title:

- Let's be a bit more descriptive:

```diff
- title = "Deploy using MiniKF on GCP"		
+ title = "Deploy Kubeflow using MiniKF on Google Cloud"
```

LMKWYT

cc @alfsuse @RFMVasconcelos

cspavlou commented:
I replaced all occurrences of "GCP" with "Google Cloud". I also updated some screenshots to to reflect the latest MiniKF version on Google Cloud. Changed the commit message accordingly

alfsuse commented:
/lgtm

8bitmp3 commented:
/assign @animeshsingh @Bobgy @joeliedtke @rmgogogo

PTAL and `/approve` when OK, thank you 👍

Bobgy commented:
Thank you @cspavlou for the update and @8bitmp3 for the review!
/approve
All looking good to me!
